relay-server/src/services/processor.rs

```diff
@@ -2658,25 +2658,24 @@ pub struct DynamicQuotas<'a> {
 
 impl<'a> DynamicQuotas<'a> {
     /// Returns a new [`DynamicQuotas`]
-    pub fn new(global_config: &'a GlobalConfig, project_quotas: &'a [Quota]) -> Self {
```

Comment:
These can stay public, just the type shouldn't be public (so no one can import it and use it by accident).

Reply:
hm, how can the constructors be public but the type not? https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=9f89537d6f8d9660062c1df5b6eec9e5